src/gsi/read_satwnd.f90

```diff
@@ -690,20 +689,19 @@ subroutine read_satwnd(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,sis
               call ufbint(lunin,obsdat,4,1,iret,obstr_v1)
            endif
 
-! run qc if the ob has qc information
-
+           ! reject data with missing pressure or wind
            ppb=obsdat(2)
-           if (ppb > 100000000.0_r_kind .or. &
-               hdrdat(3) >100000000.0_r_kind .or. &
-               obsdat(4) > 100000000.0_r_kind) cycle loop_readsb
-           if(ppb >r10000) ppb=ppb/r100
+           if (ppb>rmiss .or.  hdrdat(3)>rmiss .or.  obsdat(4)>rmiss) cycle loop_readsb
+           ppb=ppb/r100
```

Comment:
A pressure value range check was removed here:  **if(ppb >r10000)**
I think it's a good idea to keep it, in case a data provider changes the pressure units.

Reply:
@jack-woollen I agree with Iliana, I think this is the only remaining issue for satwnd and I can approve once this check is reinstated (or if it was included elsewhere and I missed it, let me know)

Reply:
@ilianagenkova @BrettHoover-NOAA  How about if I add a comment such as

 if(ppb >r10000) ppb=ppb/r100 ! ppb<10000 may indicate data reported in hPa

Although it seems like a realistic cutoff might be ppb>r1000

Reply:
I think this would work, the original limit is probably better for keeping the check completely unambiguous about realistic Pa/hPa values

Reply:
@BrettHoover-NOAA Okay! I'll leave it at that. Though I have to say, haven't seen much 10K hPa data! Let's say the data can be Pa/daPa/hpa.

Reply:
@jack-woollen , adding that comment is very helpful, especially for users who don't have experience with retrievals and what goes on data provider's end. Thanks!